bin/gce-delete-instance

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
 
-uage="Usage: `basename $0` <instance_name> [-q | --quite]"
+delete-instance() {
+  uage="Usage: `basename $0` <instance_name> [-q | --quite]"
```

Comment:
The usage string should be combined with the `--all` case.  I would need you to help me fix the typo (`usage` instead of `uage`).